hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieMultiTableDeltaStreamer.java

```diff
@@ -117,7 +117,9 @@ private void populateTableExecutionContextList(TypedProperties properties, Strin
       checkIfTableConfigFileExists(configFolder, fs, configFilePath);
       TypedProperties tableProperties = UtilHelpers.readConfig(fs, new Path(configFilePath), new ArrayList<>()).getConfig();
       properties.forEach((k, v) -> {
-        tableProperties.setProperty(k.toString(), v.toString());
+        if (tableProperties.get(k) == null) {
```

Comment:
Can you add a check for null and empty value as well?

Reply:
I think we don't need to check for empty,because some properties allow to empty such as `hoodie.datasource.write.partitionpath.field`